ui/apps/platform/src/Containers/AccessControl/AuthProviders/AuthProviders.tsx

```diff
@@ -195,6 +215,53 @@ function AuthProviders(): ReactElement {
                     }
                 />
             )}
+            <PageSection>
+                <Alert
+                    isInline
+                    variant="info"
+                    title={
+                        <span>
+                            Consider using short-lived tokens for machine to machine communications
+                            such as CI/CD pipelines, scripts, and other automation.
+                        </span>
+                    }
+                >
+                    <Flex direction={{ default: 'row' }}>
+                        <ExternalLink>
+                            <a
+                                href={getVersionedDocs(
+                                    version,
+                                    'operating/manage-user-access/configure-short-lived-access.html#configure-short-lived-access'
+                                )}
+                                target="_blank"
+                                rel="noopener noreferrer"
+                            >
+                                How to configure short-lived access
```

Comment:
What are pro and con for this phrase in context versus the actual heading in the linked doc?
* **How to configure** short-lived access
* **Configuring** short-lived access

Reply:
I wanted to distinguish it from the actual link to the machine access configuration page. While there's the "external" link icon, wanted to make it more clear from the text that it is a link to a documentation.